src/guides/v2.4/graphql/mutations/create-gift-registry.md

```diff
@@ -0,0 +1,290 @@
+                  "label": "bridal_registry",
+                  "dynamic_attributes_metadata": [
+                       {
+                        "code": 1,
+                        "input_type": "sample_input_type",
+                        "attribute_group": "att_group",
+                        "label": "metadata of dynamic attributes",
+                        "is_required": true,
+                        "sort_order": 0
+                       } ]
+                  },
+              "message": "My wedding message",
+              "created_on": "22 September 2020",
+              "privacy_settings": "PRIVATE",
+              "status": "ACTIVE",
+              "owner_name": "Owner",
+              "registrants": [
+                  {
+                      "id": 1,
+                      "first_name": "First",
+                      "last_name": "Last",
+                      "email": "first@mail.com",
+                      "dynamic_attributes": [
+                          {
+                              "code": "role",
+                              "value": "bride"
+                          }
+                      ]
+                  },
+                  {
+                      "first_name": "Second",
+                      "last_name": "Last 2",
+                      "email": "second@mail.com",
+                      "dynamic_attributes": [
+                          {
+                              "code": "role",
+                              "value": "partner"
+                          }
+                      ]
+                  }
+              ],
+              "shipping_address": {
+                   "address_id": 1
+              },
+              "dynamic_attributes": [
+                  {
+                      "code": "number_of_guests",
+                      "value": "101"
+                  }, {
+                      "code": "event_date",
+                      "value": "2022-12-12"
+                  }, {
+                      "code": "event_country",
+                      "value": "MD"
+                }, {
+                      "code": "event_location",
+                      "value": "Unknown"
+                }
+              ],
+              "items": [
+                 {
+                   "id": "P1",
+                   "quantity": 2,
+                   "quantity_fulfilled": 2,
+                   "added_on": "22 September 2020"
+                 },
+                 {
+                   "id": "P2",
+                   "quantity": 4,
+                   "quantity_fulfilled": 2,
+                   "added_on": "22 September 2020"
+                 }
+              ]
+              }
+        }
+     }
+  }
+```
+
+## Input attributes
+
+The `CreateGiftRegistryInput` input object defines the gift registry.
+
+### CreateGiftRegistryInput attributes {#CreateGiftRegistryInput}
+
+The `CreateGiftRegistryInput` object contains the following attributes:
+
+Attribute |  Data Type | Description
```

Comment:
Please sort the attributes alphabetically by the attribute name.

Reply:
I have sorted it